sky/serve/serve_state.py

```diff
@@ -43,7 +44,20 @@ def create_table(cursor: 'sqlite3.Cursor', conn: 'sqlite3.Connection') -> None:
         replica_id INTEGER,
         replica_info BLOB,
         PRIMARY KEY (service_name, replica_id))""")
-
+    cursor.execute("""\
+        CREATE TABLE IF NOT EXISTS versions (
+            service_name TEXT,
+            version INTEGER,
+            spec BLOB,
+            yaml STR,
+            PRIMARY KEY (service_name, version))""")
+    # For backward compatibility.
+    # TODO(MaoZiming): Remove this function after all users have migrated to
+    # the latest version of SkyServe.
+    # Add version column to services table
+    db_utils.add_column_to_table(
+        cursor, conn, 'services', 'version',
+        f'INTEGER DEFAULT {constants.INITIAL_VERSION}')
```

Comment:
nit: reorder so `db_utils.add_column_to_table` is attached to precious create table

Reply:
what do you mean by  attached to precious (previous?) create table

Reply:
Oh nvm; feeling like we should add the `db_utils.add_column_to_table` after L51-52 here: https://github.com/skypilot-org/skypilot/blob/b59ab222dfb8a30ba34baf8f062a8bbc50145821/sky/serve/serve_state.py#L50-L52